out/webkitgtk-sharp.dll.config

```diff
@@ -1,3 +1,6 @@
 <configuration>
   <dllmap dll="webkitgtk-3.0" target="libwebkitgtk-3.0.so.0"/>
+  <dllmap dll="soup-2.4" target="libsoup-2.4.so.1"/>
+  <dllmap dll="libgtk-3-0.dll" target="libgtk-3.so.0"/>
+  <dllmap dll="libwebkitgtksharpglue-1.10.2.so" target="libwebkitgtksharpglue-1.10.2.so"/>
```

Comment:
Is this needed? The pinvoke loader will use the original name if all else fails, not sure why this is hardcoded here (and when we change glue versions, this will become useless)